clickhouse/src/types/value_ref.rs

```diff
@@ -50,7 +50,7 @@ pub enum ValueRef<'a> {
     String(&'a [u8]),
     Float32(f32),
     Float64(f64),
-    Date(u16, Tz),
+    Date(u16),
```

Comment:
Date<Tz> will be deprecated and I'm using NaiveDate instead, so I've removed the TimeZone related information, not sure if this is appropriate.

Reply:
It'ok, `Date` is u16 in clickhouse's protocol.